redash/settings/dynamic_settings.py

```diff
@@ -0,0 +1,6 @@
+import os
+from .helpers import int_or_none
+
+# Replace this method with your own implementation in case you want to limit the time limit on certain queries or users.
+def get_query_time_limit(is_scheduled, user_id):
```

Comment:
I thought we have a `User` object at hand when calling this. Considering we don't, let's pass in `org_id` as well (or only `org_id`?).

Reply:
If the purpose of extracting this to a dynamic class was to allow users to customize this according to their needs, I'd say a `user_id` would allow more cases to be handled. I say we provide both.